server/api/diagnostic_test.go

```diff
@@ -82,17 +82,20 @@ func (suite *diagnosticTestSuite) TestSchedulerDiagnosticAPI() {
 
 	suite.NoError(tu.ReadGetJSON(re, testDialClient, addr, cfg))
 	suite.False(cfg.Schedule.EnableDiagnostic)
+	suite.Equal(float64(0), cfg.Schedule.OperatorTimeoutOffset.Seconds())
 
 	ms := map[string]interface{}{
-		"enable-diagnostic": "true",
-		"max-replicas":      1,
+		"enable-diagnostic":       "true",
+		"operator-timeout-offset": "10s",
```

Comment:
Is it necessary to add new config? Maybe we can dynamic control offset by heartbeat interval

Reply:
The duration is between the tikv send and the pd begin to handle the heartbeat request. The interval will be reset before tikv sends the request. so the interval has no relation of the handler duration.